spk/aria2/Makefile

```diff
@@ -0,0 +1,24 @@
+SPK_NAME = aria2
+SPK_VERS = 1.31.0
+SPK_REV = 1
+SPK_ICON = src/aria2.png
+
+DEPENDS = cross/$(SPK_NAME)
+
+MAINTAINER = SynoCommunity
+DESCRIPTION = aria2 is a lightweight multi-protocol and multi-source command-line download utility.
+RELOAD_UI = no
+DISPLAY_NAME = aria2
+STARTABLE = no
+BETA = 0
+CHANGELOG = Initial version
+
+HOMEPAGE   = http://aria2.github.io
+LICENSE    = GPLv2/GPLv3
+
+INSTALLER_SCRIPT = src/installer.sh
+SSS_SCRIPT       = src/dsm-control.sh
```

Comment:
Please remove `SSS_SCRIPT` and `dsm-control.sh` file as application does not run as a service